tensorflow_transform/mappers.py

```diff
@@ -120,7 +120,8 @@ def scale_by_min_max(x,
     if output_min >= output_max:
       raise ValueError('output_min must be less than output_max')
 
-    x = tf.to_float(x)
+    x = tf.dtypes.cast(x, 'float64', name='Cast_Float')
```

Comment:
Please use tf.cast instead of tf.dtypes.cast and remove the name kwarg from all of these call sites for consistency with the rest of the code base.

Please also use tf.dtypes instead of string literals (i.e. tf.float64 instead of 'float64').